test/onnx/test_fx_op_consistency.py

```diff
@@ -451,8 +467,8 @@
         "nn.functional.dropout",
         reason=onnx_test_common.reason_dynamo_does_not_support("Dropout"),
     ),
-    xfail(
-        "nn.functional.elu",
+    skip(
+        "nn.functional.elu",  # SegFault when ORT < 1.15
         dtypes=(torch.float64,),
```

Comment:
Also ORT doesn't claim to support float64 in most cases so we may be able to exclude float64 entirely by removing it from the tested types.

Reply:
Done